clang/include/clang/Basic/arm_sve.td

```diff
@@ -2148,6 +2148,11 @@ let TargetGuard = "sme2" in {
   def SVSCLAMP_X4 : SInst<"svclamp[_single_{d}_x4]",  "44dd",   "csil",     MergeNone, "aarch64_sve_sclamp_single_x4",  [IsStreaming], []>;
   def SVUCLAMP_X4 : SInst<"svclamp[_single_{d}_x4]",  "44dd",   "UcUsUiUl", MergeNone, "aarch64_sve_uclamp_single_x4",  [IsStreaming], []>;
   def SVFCLAMP_X4 : SInst<"svclamp[_single_{d}_x4]",  "44dd",   "hfd",      MergeNone, "aarch64_sve_fclamp_single_x4",  [IsStreaming], []>;
+
+  let TargetGuard = "b16b16"in {
```

Comment:
Are you sure this work 
 TargetGuard = "sme2" in {
     TargetGuard = "b16b16" in {
I have a distant memory  saying it does not work, that we need 
 TargetGuard = "sme2,b16b16" in {

Reply:
I does not work. A test for target features (in `Sema`) would catch that.

Reply:
Fixed. Also added SEMA test for this